go.mod

```diff
@@ -794,8 +829,16 @@ exclude (
 	github.com/knadh/koanf/providers/confmap v0.1.0
 )
 
-// Temporarely use forks of trivy libraries to use lazy initialization of zap loggers.
-// Patch was pushed upstream but maintainers would prefer moving to slog once 1.22 is out
-replace github.com/aquasecurity/go-dep-parser => github.com/lebauce/go-dep-parser v0.0.0-20231030190658-51bbb436f1b8
-
-replace github.com/aquasecurity/trivy-db => github.com/lebauce/trivy-db v0.0.0-20231031000443-2d41e0fab0ae
+replace (
+	// Stick to v0.27.6 as bumping client-go to 0.28.x breaks cluster agent leader election
+	k8s.io/api => k8s.io/api v0.27.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.27.6
+	k8s.io/apiserver => k8s.io/apiserver v0.27.6
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.6
+	k8s.io/client-go => k8s.io/client-go v0.27.6
+	k8s.io/component-base => k8s.io/component-base v0.27.6
+	// Small compilation fixes so that k8s.io/apiserver can compile against kube-openapi
+	k8s.io/kube-openapi => github.com/datadog/kube-openapi v0.0.0-20231101162351-39b249f0ed92
```

Comment:
I'm still wondering if it wasn't worth to update our code to use the new kube-openapi version instead of forking this repo, because it will make later other update more difficult 

the diff from https://github.com/kubernetes/kube-openapi/compare/master...DataDog:kube-openapi:restore-register-service-error doesn't look big so we can do the removal of this change in another PR